lib/WebService/Stripe.pm

```diff
@@ -70,6 +70,10 @@ method get_balance(:$headers) {
     return $self->get( "/v1/balance", {}, headers => $headers );
 }
 
+method get_balance_history(:$headers) {
```

Comment:
Also, I think this should be renamed 'get_balance_transactions' because Stripe references this as 'balance_transaction' when you get charge objects, etc. And then we should also have a 'get_balance_transaction' call that takes a single ID, also:  /v1/balance/history/:id